src/react-components/2d-hud.js

```diff
@@ -1,16 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
-import queryString from "query-string";
 
 import styles from "../assets/stylesheets/2d-hud.scss";
 
 import FontAwesomeIcon from "@fortawesome/react-fontawesome";
 import faPlus from "@fortawesome/fontawesome-free-solid/faPlus";
 
-const qs = queryString.parse(location.search);
+const qs = new URLSearchParams(location.search);
 function qsTruthy(param) {
```

Comment:
I see this duplicated in [`hub.js`](https://github.com/mozilla/hubs/pull/466/files#diff-36e9875da4ba69fe26a79e004718180cR132). could put the method in a common utils file?

Reply:
I think this is a weird and unpredictable technique that I would rather just get rid of, personally, so I'm hesitant to enshrine it somewhere. (I would prefer to declare that the query parameter is truthy if it is present at all, which seems simpler.)